packages/devtools_app/lib/src/shared/table_data.dart

```diff
@@ -61,6 +61,8 @@ abstract class ColumnData<T> {
   String getDisplayValue(T dataObject) =>
       getValue(dataObject)?.toString() ?? '';
 
+  String? getCaption(T dataObject) => '';
```

Comment:
Switch to returning `null` by default instead of an empty string (that way the check `if (column.getCaption(node) != null)` will work as expected).